.travis.yml

```diff
@@ -0,0 +1,26 @@
+# Documentation: http://docs.travis-ci.com/user/languages/julia/
+language: julia
+os:
+  - linux
+dist: xenial
+if: branch = master OR tag IS present OR type = pull_request
+julia:
+  - 1.3
+  - 1.4
+  - 1.5
+notifications:
+  email: false
+script:
+ - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
+ - travis_wait julia -e 'using Pkg; Pkg.build("K8sClusterManagers"); Pkg.test("K8sClusterManagers"; coverage=true)'
```

Comment:
In Lighthouse, we have these commented and then an `after_success` step for coverage: https://github.com/beacon-biosignals/Lighthouse.jl/blob/4ea93408df1b6eb1412900bdaa99a52da4b95c15/.travis.yml

I am not super familiar with Travis so I'm not sure the difference, but I'd probably follow Lighthouse to be safe

Reply:
I just copied Lighthouse's travis.yml, minus an unused env variable. Seems to build in CI.